CODEOWNERS

```diff
@@ -0,0 +1,6 @@
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
```

Comment:
So does this mean it would actually request both of us as reviewers for any pull request?

Reply:
Yep! This means that we will both be added as reviewers. 
As a follow-up to this change I'll configure our github workflow so that only one approval is neccessary.

Reply:
https://docs.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests